packages/feedback/typings/FeedbackForm.d.ts

```diff
@@ -8,15 +8,16 @@ type AboutOption = {
     label?: React.ReactNode;
 };
 
-export interface FeedBackFormProps {
+export interface FeedbackFormProps {
     name: string;
     onFeedbackSent?: (feedback?: {[key:string]:any}) => void;
     faceOptions?: FaceOption[];
     aboutOptions?: AboutOption[];
     prompt?: string;
     staticFields?: object;
+    modalHeaderProps?: object;
```

Comment:
Let's add `@types/reactstrap` to the dev deps of this package and then use the props to extend here.

ex.
```js
yarn add @types/reactstrap
...
import { ModalHeaderProps } from 'reactstrap'
...
modalHeaderProps?: ModalHeaderProps;
```